lib/src/test/java/com/thenewboston/api/bankapi/repository/BankRepositoryTest.kt

```diff
@@ -19,21 +21,23 @@ import io.mockk.coVerify
 import io.mockk.impl.annotations.MockK
 import kotlinx.coroutines.ExperimentalCoroutinesApi
 import kotlinx.coroutines.test.runBlockingTest
-import org.junit.jupiter.api.BeforeEach
 import org.junit.jupiter.api.Test
 import java.io.IOException
 import javax.xml.validation.Validator
+import org.junit.jupiter.api.BeforeAll
+import org.junit.jupiter.api.TestInstance
 
 @ExperimentalCoroutinesApi
 @KtorExperimentalAPI
+@TestInstance(TestInstance.Lifecycle.PER_CLASS)
 class BankRepositoryTest {
 
     @MockK
     lateinit var bankDataSource: BankDataSource
 
     private lateinit var repository: BankRepository
 
-    @BeforeEach
+    @BeforeAll
```

Comment:
This little change cut the test time of the entire file from ~1500ms to ~500ms for me :smile: